migrationmanager/migrators/logs/migrations/000009_add_usage.up.sql

```diff
@@ -0,0 +1,15 @@
+--we don't have timestamp in the order by field as we need to update it continuously.
+CREATE TABLE IF NOT EXISTS signoz_logs.usage ON CLUSTER {{ .SIGNOZ_CLUSTER }} (
+    tenant String,
+    collector_id String,
+    exporter_id String,
+    timestamp DateTime,
+    data String
+) ENGINE MergeTree()
+ORDER BY (tenant, collector_id, exporter_id, timestamp)
```

Comment:
order by has the timestamp by the comment above says otherwise.

Reply:
Aah that was a comment which was added by mistake due to previous implementations in usage.go and was not removed, I copied it to the migrations without checking, have removed it.